server/__init__.py

```diff
@@ -125,21 +118,14 @@ def validation_func(lobby_conn):
                 def validation_func(lobby_conn):
```

Comment:
Dame here, extract functions out of the scope, just define them once

Reply:
No point. These capture the current `game` variable. If you define this function elsewhere you would just have to call it with `functools.partial` which would probably be worse. Although again, defining 1 function per second is going to be completely negligible.